<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlabDeleteObjectDialog</class>
 <widget class="QDialog" name="SlabDeleteObjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>241</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBoxObjectType">
      <item>
       <property name="text">
        <string>Surfaces</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Line Supports</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Surface Supports</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Line Loads</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nodal Loads</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Surface Loads</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
